src/core/lib/security/credentials/external/file_external_account_credentials.h

```diff
@@ -52,6 +54,8 @@ class FileExternalAccountCredentials final : public ExternalAccountCredentials {
   std::string file_;
   std::string format_type_;
   std::string format_subject_token_field_name_;
+  // Used to lock the file loading.
+  gpr_mu mu_;
```

Comment:
Instead of using `gpr_mu` and `gpr_mu_lock()` / `gpr_mu_unlock()`, please use `grpc_core::Mutex` and `grpc_core::MutexLock`.